src/test/java/br/com/pucgo/perguntaai/PerguntaaiBackendApplicationTests.java

```diff
@@ -1,27 +1,49 @@
 package br.com.pucgo.perguntaai;
 
+import br.com.pucgo.perguntaai.controllers.v1.UserController;
+import br.com.pucgo.perguntaai.models.DTO.UserDto;
 import br.com.pucgo.perguntaai.models.User;
+import br.com.pucgo.perguntaai.models.form.UserForm;
 import br.com.pucgo.perguntaai.repositories.UserRepository;
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
 
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
 
 @SpringBootTest
```

Comment:
é necessário dividir os testes em pacotes específicos, de modo a manter tudo organizado desde o ínicio.

Ou seja, uma sugestão é criar um pacote `"controllers"` dentro de `test.java.br.com.pucgo.perguntaai`, sendo que para toda operação REST que você fizer, deve-se criar uma classe de teste específica. 

Ou seja, se for testar o método de **cadastrar o usuário**, pode-se criar uma classe só para este caso de uso ou crie uma classe dentro do pacote `controllers` com todos os testes de CRUD associados ao User. Eu recomendo a primeira abordagem. Mas caso escolha a segunda, o PR será aceito.